utils/src/main/scala/com/salesforce/op/utils/io/avro/AvroInOut.scala

```diff
@@ -35,11 +35,12 @@ import java.net.URI
 import com.salesforce.op.utils.spark.RichRDD._
 import org.apache.avro.Schema
 import org.apache.avro.generic.GenericRecord
-import org.apache.avro.mapred._
+import org.apache.avro.mapred.AvroKey
```

Comment:
`mapreduce` still uses `AvroKey` from `mapred`. is this correct?

Reply:
class AvroKeyInputFormat<T> extends FileInputFormat<AvroKey<T>, NullWritable> . 
This uses AvroKey from mapred only.